Cargo.toml

```diff
@@ -49,11 +49,9 @@ resolver = "2"
 [profile.test]
 debug = 1
 opt-level = 3
-split-debuginfo = "unpacked"
```

Comment:
looks like issue was here @Mec-iS

Reply:
cool. i think Rust 1.65 dropped support for that